### ISSUE TITLE
Coding standard refers to old tools

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3414]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#3414|
|Type|enhancement|
|Created|2008-09-06 21:31:29Z|
|Branch|https://github.com/twisted/twisted/tree/old-coding-tools-3414|

It mentions the admin/accepttests and admin/change scripts that were recently removed.

<details><summary>Searchable metadata</summary>

```
trac-id__3414 3414
type__enhancement enhancement
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__branches_old_coding_tools_3414 branches/old-coding-tools-3414
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1220736689000000 1220736689000000
changetime__1220913131000000 1220913131000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

(In [24736]) Branching to 'old-coding-tools-3414'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

See r24742

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24750]) Merge old-coding-tools-3414: Remove references to old admin tools in the coding standard doc. Also removes ChangeLog section, which is now a release management task.

Author: thijs
Reviewer: exarkun
Fixes: [#3414](https://github.com/twisted/twisted/issues/3414)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
